### PR TITLE
Add DOAP file

### PR DESCRIPTION
**Pull request by ocket8888**
## What does this PR (Pull Request) do?
- [x] This PR is not related to any Issue

This PR adds a DOAP (Description of a Project) file to the repository that will enable the Traffic Control project to be listed on [projects.apache.org](https://projects.apache.org/).

## Which Traffic Control components are affected by this PR?
None

## What is the best way to verify this PR?
Verify the DOAP is both valid XML and RDF, verify that it is accurate and follows Apache's guidelines for DOAP files.

## The following criteria are ALL met by this PR
- [x] Tests are unnecessary
- [x] Documentation is unnecessary
- [x] An update to CHANGELOG.md is not necessary
- [x] This PR includes any and all required license headers
- [x] This PR **DOES NOT FIX A SERIOUS SECURITY VULNERABILITY**

### CODE REVIEW

File: doap_Traffic_Control.rdf [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+				<revision>5.1.0</revision>
+			</Version>
+		</release>
+		<release>
+			<Version>
+				<name>Apache Traffic Control v5.0.0</name>
+				<created>2021-01-21</created>
+				<revision>5.0.0</revision>
+			</Version>
+		</release>
```
Comment (zrhoffman) [inline doap_Traffic_Control.rdf:68]: IMO no need to link to obsoleted releases of the latest release branch.

File: doap_Traffic_Control.rdf [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+	xmlns:foaf="http://xmlns.com/foaf/0.1/">
+<!--
+		Licensed to the Apache Software Foundation (ASF) under one or more
+		contributor license agreements.  See the NOTICE file distributed with
+		this work for additional information regarding copyright ownership.
+		The ASF licenses this file to You under the Apache License, Version 2.0
+		(the "License"); you may not use this file except in compliance with
+		the License.  You may obtain a copy of the License at
+	 
+				 https://www.apache.org/licenses/LICENSE-2.0
```
Comment (zrhoffman) [inline doap_Traffic_Control.rdf:16]: For legal reasons, [the source file header](https://www.apache.org/legal/src-headers.html#headers) still includes a HTTP link, not HTTPS. They're trying to change it, though. From [LEGAL-536](https://issues.apache.org/jira/browse/LEGAL-536):

>I agree that the text should be changed on the license (several places on the Apache site and internal documents).
>
>I also believe that a license with either the http:// or the https:// should be treated the same.
>
>But we need to be sensitive to many external entities that compare third parties' licenses with the Apache license and a small change like this is likely to cause confusion.
>
>First we need a plan to notify third parties about the change that we plan to make so they are not blindsided. 

LEGAL-536 was last updated 4 days ago, so they're actively working on this. Mailing list thread: https://lists.apache.org/thread.html/r2476215532a0ac81d7875f1e1d69c2b36e492348265ce481a5c99a6c%40%3Clegal-discuss.apache.org%3E

**Review by zrhoffman — CHANGES_REQUESTED** [review]

File: doap_Traffic_Control.rdf (hunk 1 above)
Comment (ocket8888) [reply doap_Traffic_Control.rdf:68]: I don't really care either way, I just figured if we had been doing this all along they'd have been in there

File: doap_Traffic_Control.rdf (hunk 2 above)
Comment (ocket8888) [reply doap_Traffic_Control.rdf:16]: I'm sure it's fine. That header was generated for me by Apache's own [DOAP creation tool](https://projects.apache.org/create.html).

Comment (zrhoffman) [reply doap_Traffic_Control.rdf:16]: Okay

**Review by zrhoffman — APPROVED** [review]

**Pull request merged**